tools/ci/check_stability.py

```diff
@@ -353,16 +350,15 @@ def run(venv, wpt_args, **kwargs):
                              status="failed" if inconsistent else "passed")
     else:
         logger.info("No tests run.")
+        retcode = 3
```

Comment:
Are we sure this never happens in practice? I'm slightly concerned that maybe we sometimes pass this a list of things that could be tests but actually aren't so nothing would actually run. I *think* it should be OK but there might be an edge case like a `.py` file under `webdriver` that doesn't have any `test` functions, or something.

Reply:
No you're right. It's totally possible to have `tests_changed` empty but `files_affected` not, though I'm not sure if the scenario you described would be an edge case (not familiar with `webdriver/*`).

We should dive deeper: raise an exception from `wptrunner.run_tests`.

Reply:
Sorry, going down into `wptrunner.run_tests` doesn't really solve the problem at all.

Let me be conservative and only return errors if `tests_changed` is non-empty.